pom.xml

```diff
@@ -85,7 +85,7 @@
                     <execution>
                         <id>bin</id>
                         <goals>
-                            <goal>single</goal>
```

Comment:
I think this is correct, `attached` was deprecated while ago. The issue is in the `test` goal which fails to discover tests.

Reply:
(at least on my machine)

Reply:
fixed